services/ui-src/src/components/cards/FundingSourcesEntity.test.tsx

```diff
@@ -0,0 +1,44 @@
+import { render, screen } from "@testing-library/react";
+import { testA11y } from "utils/testing/commonTests";
+import {
+  mockFundingSourcesFormattedEntityData,
+  mockFundingSourcesFormattedEntityDataNoQuarters,
+} from "utils/testing/mockEntities";
+import { FundingSourcesEntity } from "./FundingSourcesEntity";
+
+const fundingSourcesEntity = (
+  <FundingSourcesEntity
+    formattedEntityData={mockFundingSourcesFormattedEntityData}
+  />
+);
+
+const fundingSourcesEntityNoQuarters = (
+  <FundingSourcesEntity
+    formattedEntityData={mockFundingSourcesFormattedEntityDataNoQuarters}
+  />
+);
+
+describe("<FundingSourcesEntity />", () => {
+  test("FundingSourcesEntity renders correctly", () => {
```

Comment:
I'd update the descriptions for the tests to be
`renders with quarters` and `renders without quarters`

<img width="436" alt="Screenshot 2024-07-10 at 2 39 31 PM" src="https://github.com/Enterprise-CMCS/macpro-mdct-mfp/assets/169298580/37e11be2-7ded-49d9-a41b-08c2dba29c2e">